client/modules/workspace/src/AppsSideNav/AppsSideNav.tsx

```diff
@@ -1,19 +1,14 @@
 import React from 'react';
 import { Menu, MenuProps } from 'antd';
 import { NavLink } from 'react-router-dom';
-import styles from './AppsSideNav.module.css';
 import { TAppCategory, TPortalApp } from '@client/hooks';
 import { useGetAppParams } from '../utils';
 
 const AppsNavLink: React.FC<React.PropsWithChildren<{ to: string }>> = ({
   to,
   children,
 }) => {
-  return (
-    <NavLink to={to} className={styles.navLink}>
-      <div>{children}</div>
-    </NavLink>
-  );
+  return <NavLink to={to}>{children}</NavLink>;
```

Comment:
With no more `className` prop passed, we can instead get rid of the `AppsNavLink` component and just use the `NavLink` component

Reply:
Done!